nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -118,6 +117,15 @@ in
             apply = lib.concatStringsSep ",";
           };
 
+          STATIC_ROOT = mkOption {
+            type = types.str;
+            default = cfg.dataDir + "/static";
+            defaultText = lib.literalExpression ''
+              config.${opt.dataDir} + "/static"
+            '';
+            description = lib.mdDoc "State directory for dynamically generated static files.";
+          };
```

Comment:
> Wouldn't it be easier to override the package and change the files in there?

The files are generated dynamically by the manage.py script, so I fail to see how overriding would improve things here?

Reply:
This was already overridable through `healthchecks.settings` because it is being read from environment in local_setting.py and the default path was defined in the environment above. I just moved it here and documented it. Note that it was already mentioned in description for `healthchecks.settings` too.

I am happy to remove this change if you think it creates unnecessary confusion. I actually don't need it and only wanted to clean things up a bit.

Reply:
Sounds like there is a consensus that this change is not needed. Dropped this commit.